setup.py

```diff
@@ -31,6 +31,7 @@
         "price-parser>=0.3.4",
         "web-poet>=0.14.0",
         "zyte-parsers>=0.5.0",
+        "scrapy>=2.10",
```

Comment:
Could we lower this? [`global_object_name` was added in 1.4.0](https://github.com/scrapy/scrapy/commit/e285b1d6c2aaa1fdfe788f1894b0196bc64d1be1), even if it was improved later.

Reply:
Also, we need to update [tox.ini](https://github.com/zytedata/zyte-common-items/blob/c326028bfa9b779832d512c0e2c7f29a4681e91a/tox.ini#L31-L42) accordingly to make sure we test the minimum supported version as well.

Reply:
Ah, this [global_object_name](https://github.com/scrapy/scrapy/commit/e285b1d6c2aaa1fdfe788f1894b0196bc64d1be1) is not used anymore - do we need to lower it(and fix tox) in this case? And if so - to which version?

Reply:
I think we should lower the version to [2.0.1](https://github.com/scrapy-plugins/scrapy-zyte-api/blob/e8b4b11f8b94edc5ccf29be076e4f7a118097bcb/setup.py#L27) at least, if we can.

Reply:
Interesting failure in CI, unrelated to your changes other than adding scrapy as a dependency. Let me look into that myself.

Reply:
2.0.1 doesn't work - the test [test_components_request.py](https://github.com/zytedata/zyte-common-items/blob/main/tests/test_components_request.py#L65) fails. The issue relates to creating headers from multiple headers and I see that this issue was fixed in [2.7.0](https://docs.scrapy.org/en/latest/news.html#scrapy-2-7-0-2022-10-17) `Fixed Headers.getlist returning only the last header`.
I've checked 2.7.0 - all tests passed.  With 2.6.3 - the same issue. Left 2.7.0 as a min version.